common/static/common/js/discussion/views/discussion_topic_menu_view.js

```diff
@@ -45,7 +45,8 @@
             renderCategoryMap: function(map) {
                 var categoryTemplate = edx.HtmlUtils.template($('#new-post-menu-category-template').html()),
                     entryTemplate = edx.HtmlUtils.template($('#new-post-menu-entry-template').html()),
-                    mappedCategorySnippets = _.map(map.children, function(name) {
+                    mappedCategorySnippets = _.map(map.children, function(nameType) {
```

Comment:
@robrap I have addressed the comments. Kindly review again.
Also, I have updated the sandbox.